gen_import_utils.py

```diff
@@ -5,13 +5,61 @@
 import sys
 import numpy as np
 import pandas as pd
-import json
 import hmac
 import settings
 import os
-
+from string_utils import remove_non_numerics
 # import list tools
 
+
+def separate_titles(row):
+    first_name_titles = ['Dr.', 'Mrs.', 'Mr', 'Mr.', 'Fr.', 'Miss.', 'Sister', 'Bro.',
+                         'Frère', 'Captain', "M. l'Abbé", 'Miss', 'Prof.',
+                         'M.', 'Père', 'Abb.', '(Mrs.)', "l'Abbé", 'Ranger',
+                         'abb.', 'Farm Advisor', 'Ma.', 'Rev. Père', 'Abbé',
+                         'Rev.', 'Mr. and Mrs.', 'Dr. sc.', 'Fr', 'Mrs', 'abbé', 'Ed.',
+                         'F.', 'Mr. & Mrs.', 'Professor', 'Brother', 'Frère/Brother', 'Sgto.',
+                         'Frè', 'der Lehrer [= the teacher]', 'Botany Volunteer', 'M. [= Monsieur]',
+                         'DR.', 'Prof. Dr.', 'Capt.', 'Col.', 'Lt. M.D.', 'Major', 'Ms.', 'Profr.',
+                         'Pr.', 'Lt.', 'Father', 'Baron', 'Fre.', 'Fre.']
+
+    last_name_titles = ['M.D', 'M.D.', 'Esq.', 'III', 'Jr.', 'A.M.', 'JR', 'Sr.']
+
+    # Initialize Title as empty
+    row['Title'] = ''
+
+    # Check and process first name titles
+    for title in first_name_titles:
+        if row['First Name'].startswith(title + " "):
+            row['Title'] = title
+            row['First Name'] = row['First Name'][len(title)+1:]
+            break  # Assuming only one title, break after finding
+
+    # Check and process last name titles
+    for title in last_name_titles:
+        if row['Last Name'].endswith(" " + title):
+            row['Title'] = title
+            row['Last Name'] = row['Last Name'][:-(len(title)+1)]
+            break
+
+    return row
+
+
```

Comment:
used to combine date columns into one properly formatted date column according to casbotany db standards. Used in picturae_batch_wrangler, but added here to have parallel changes across util files.